pkg/httperrors/httperrors.go

```diff
@@ -122,3 +123,22 @@ func HTTPError(c echo.Context, statusCode int, message string) (e error) {
 
 	return c.JSON(statusCode, resBody)
 }
+
+// GetHTTPResponseFromErr returns the mapped http error to the generic errors model.
+func GetHTTPResponseFromErr(c echo.Context, err error) (e error) {
+
+	switch err {
+	case models.ErrInternalServerError:
+		return InternalServerError(c, err.Error())
+	case models.ErrNotFound:
+		return NotFound(c, err.Error())
+	case models.ErrConflict:
+		return Conflict(c, err.Error())
+	case models.ErrBadParamInput:
+		return BadRequest(c, err.Error())
+	case models.ErrUnauthorized:
+		return Unauthorized(c, err.Error())
+	default:
+		return InternalServerError(c, "")
```

Comment:
Should not be better here return the error message instead of "" ? If we keep in this way, it will return Internal Server Error 500 without any information which could be helpful to understand the issue. Wdyt?

Reply:
It will end up returning a message. If a message is not supplied [1], it will use the message from the status code map[2]. I did not want to duplicate the text and have to pass the string argument.

[1] https://github.com/aerogear/mobile-security-service/blob/4e7f7c8a890edad98e35c2d07bf4bde64ac9815d/pkg/httperrors/httperrors.go#L115
[2] https://github.com/aerogear/mobile-security-service/blob/4e7f7c8a890edad98e35c2d07bf4bde64ac9815d/pkg/httperrors/httperrors.go#L39

Reply:
However, should we not return something as err.getMessage() here instead of "" in the following line? 

```nodejs
return InternalServerError(c, "")
```
I mean, for example, the issue is with the db connection, so the error message will be something as `connection refuse`. Another example could be something like `number exception format`. We could return this message to clarify it in order to be easier to troubleshooting it. Wdyt?

Reply:
Line 142 is just the default in the switch statement, if we have called this function with an unknown model error. This should never happen. But as the default, I am returning a 500, just so there is a default.